src/exml_query.erl

```diff
@@ -118,17 +127,20 @@ child_with_ns([#xmlel{} = Element | Rest], NS, Default) ->
 child_with_ns([_ | Rest], NS, Default) ->
     child_with_ns(Rest, NS, Default).
 
--spec subelement_with_attr(exml:element(), AttrName :: binary(), AttrValue :: binary()) ->
-    exml:element() | undefined.
+-spec subelement_with_attr(exml:element(),
+                           AttrName :: binary(),
+                           AttrValue :: binary()) -> exml:element() | undefined.
 subelement_with_attr(Element, AttrName, AttrValue) ->
     subelement_with_attr(Element, AttrName, AttrValue, undefined).
 
--spec subelement_with_attr(Element, AttrName, AttrValue, Other) -> SubElement | Other when
-      Element :: exml:element(),
-      AttrName :: binary(),
-      AttrValue :: binary(),
-      SubElement :: exml:element(),
-      Other :: term().
+-spec subelement_with_attr(Element, AttrName, AttrValue, Other) -> SubElement |
+                                                                   Other when Element ::
```

Comment:
Is it possible to have the part in `when` formatted starting from new line, as it was done before? Current rules makes formatting awkward in this case.